MAPL_Base/MAPL_Generic.F90

```diff
@@ -4997,7 +4997,7 @@ subroutine MAPL_GenericStateClockOff(STATE,NAME,RC)
     !EOPI
 
     character(len=ESMF_MAXSTR), parameter :: IAm = "MAPL_GenericStateClockOff"
-    integer :: STATUS, n
+    integer :: STATUS !, n
```

Comment:
If "n" is no longer needed, it really should be deleted.   Or do you expect another sweep through the code after this review.

Reply:
"n" is no longer needed. But there was already commented section below the declaration which uses "n". My intent was to make it easier if anyone want to go back and uncomment that particular section to have a "ready" declaration staement